.github/workflows/ai-revision.yaml

```
@@ -0,0 +1,54 @@
+name: ai-revision
+on:
+  workflow_dispatch:
+    inputs:
+      branch:
+        description: 'Branch to revise'
+        required: true
+        type: string
+        default: 'main'
+      file_names:
+        description: 'File names to revise'
+        required: false
+        type: string
+        default: ''
+      model:
+        description: 'Language model'
+        required: true
+        type: string
+        default: 'text-davinci-003'
+      branch_name:
+        description: 'Output branch'
+        required: true
+        type: string
+        default: 'ai-revision-davinci'
+
+jobs:
+  ai-revise:
+    name: GPT Revise
```

```suggestion
    name: AI Revise
```
For consistency, everything else is now referred to as AI instead of GPT